### ISSUE TITLE
Pool - README.md

### ISSUE THREAD

**Issue by prographo** [project-serum/serum-dex#45]
Describe in plain english what it is and what it does.

**Comment by armaniferrante** [comment]
In this documentation, we should describe the expectation/different strategies for initializing the initial token supply. This has popped up a couple of times on Discord as a point of confusion.

Right now there are three options:

* Initialize the pool token supply before initializing the pool
* Initialize the pool token supply on pool initialization
* Override trait methods (e.g. `process_creation`) to explicitly check for token supply == 0 and act accordingly, which as @garywang points out, allows someone to reset the basket by fully redeeming and recreating, which may or may not be acceptable depending on the pool.

**Comment by armaniferrante** [comment]
I don't think there are plans for continued work on the pool program as written. If I'm mistaken please re-open.
